packages/material-ui/src/Backdrop/Backdrop.js

```diff
@@ -39,21 +66,23 @@ const Backdrop = React.forwardRef(function Backdrop(props, ref) {
     ...other
   } = props;
 
+  const styleProps = {
+    ...other,
```

Comment:
```suggestion
    ...props,
```

`styleProps` should contain all **props** + their defaults